webui/react/src/pages/WorkspaceDetails/WorkspaceProjects.tsx

```diff
@@ -403,43 +405,45 @@ const WorkspaceProjects: React.FC<Props> = ({ workspace, id, pageRef }) => {
 
   return (
     <>
-      <Columns page>
-        <Column>
-          <Select value={settings.whose} width={160} onSelect={handleViewSelect}>
-            <Option value={WhoseProjects.All}>All Projects</Option>
-            <Option value={WhoseProjects.Mine}>My Projects</Option>
-            <Option value={WhoseProjects.Others}>Others&apos; Projects</Option>
-          </Select>
-        </Column>
-        <Column align="right">
-          <Space wrap>
-            {!workspace.archived && (
-              <Toggle
-                checked={settings.archived}
-                label="Show Archived"
-                onChange={switchShowArchived}
-              />
-            )}
-            <Select value={settings.sortKey} width={170} onSelect={handleSortSelect}>
-              <Option value={V1GetWorkspaceProjectsRequestSortBy.NAME}>Alphabetical</Option>
-              <Option value={V1GetWorkspaceProjectsRequestSortBy.LASTEXPERIMENTSTARTTIME}>
-                Last Updated
-              </Option>
-              <Option value={V1GetWorkspaceProjectsRequestSortBy.CREATIONTIME}>
-                Newest to Oldest
-              </Option>
+      <Section>
+        <Row wrap>
+          <Column>
+            <Select value={settings.whose} width={160} onSelect={handleViewSelect}>
+              <Option value={WhoseProjects.All}>All Projects</Option>
+              <Option value={WhoseProjects.Mine}>My Projects</Option>
+              <Option value={WhoseProjects.Others}>Others&apos; Projects</Option>
             </Select>
-            {settings && <GridListRadioGroup value={settings.view} onChange={handleViewChange} />}
-            <div className={css.headerButton}>
-              {!workspace.immutable &&
-                !workspace.archived &&
-                canCreateProject({ workspace: workspace }) && (
-                  <Button onClick={handleProjectCreateClick}>New Project</Button>
-                )}
-            </div>
-          </Space>
-        </Column>
-      </Columns>
+          </Column>
+          <Column align="right">
```

Comment:
fix: again, this alignment isn't working
![Screenshot 2023-11-14 at 15-32-12 Determined](https://github.com/determined-ai/determined/assets/15078396/323ca6ce-ce2c-4d37-acd5-32e64b24b3e0)